beacon_node/store/src/hot_cold_store.rs

```diff
@@ -1667,6 +1688,137 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
         );
         Ok(())
     }
+
+    pub fn try_prune_blobs(&self, force: bool) -> Result<(), Error> {
+        let mut blob_info: BlobInfo;
+
+        match self.get_blob_info() {
+            Some(old_blob_info) => {
+                blob_info = old_blob_info;
+            }
+            None => {
+                return Ok(());
+            }
+        }
```

Comment:
Just a minor style thing, but usually we avoid `let`s with no initializer:

```suggestion
        let blob_info = match self.get_blob_info() {
            Some(old_blob_info) => {
                old_blob_info
            }
            None => {
                return Ok(());
            }
        };
```